testing/gimmeproj/main.go

```diff
@@ -199,6 +207,16 @@ func lease(ctx context.Context, duration string) error {
 	return nil
 }
 
+func list(ctx context.Context) error {
+	return withPool(ctx, func(pool *Pool) error {
+		for _, proj := range pool.Projects {
+			fmt.Println(proj.ID)
+		}
```

Comment:
if facilitating scripting, specifically, add a `-json` or `-output=json` flag.

would adding that to "status" work fine?